dev/AutoSuggestBox/AutoSuggestBox_themeresources.xaml

```diff
@@ -355,11 +363,12 @@
                             contract7Present:CornerRadius="{TemplateBinding CornerRadius}"
                             Style="{StaticResource QueryButtonStyle}"
                             BorderThickness="{TemplateBinding BorderThickness}"
-                            Padding="{ThemeResource HelperButtonThemePadding}"
+                            Padding="5,3,5,3"
```

Comment:
Should we create a new theme resources ? or fix the animation so it does not need a different padding ?